quaint/src/visitor/sqlite.rs

```diff
@@ -282,8 +282,21 @@ impl<'a> Visitor<'a> for Sqlite<'a> {
         })
     }
 
-    fn visit_json_extract(&mut self, _json_extract: JsonExtract<'a>) -> visitor::Result {
-        unimplemented!("JSON filtering is not yet supported on SQLite")
+    #[cfg(any(feature = "postgresql", feature = "mysql", feature = "sqlite"))]
+    fn visit_json_extract(&mut self, json_extract: JsonExtract<'a>) -> visitor::Result {
+        self.visit_expression(*json_extract.column)?;
+
+        if json_extract.extract_as_string {
+            self.write("->>")?;
+        } else {
+            self.write("->")?;
+        }
+        match json_extract.path.clone() {
```

Comment:
```suggestion
        }
  
        match json_extract.path.clone() {
```

Besides, is the clone necessary?

Reply:
removed